### PR TITLE
Have avify4mtest get temp dir path from env var

### PR DESCRIPTION
**Pull request by wantehchang**

### CODE REVIEW

File: tests/avify4mtest.c [hunk 1]
```diff
@@ -158,14 +158,15 @@ static avifBool encodeDecodeY4m(uint32_t width,
 
 //------------------------------------------------------------------------------
 
-int main(int argc, char * argv[])
+int main(void)
 {
-    if (argc != 2 || !strlen(argv[1])) {
-        fprintf(stderr, "Missing temporary folder path argument\n");
+    const char * testTmpdir = getenv("TEST_TMPDIR");
```
Comment (wantehchang) [inline tests/avify4mtest.c:163]: Using the hardcoded environment variable name "TEST_TMPDIR" optimizes for the Bazel and Blaze build environments.

We probably should use our own environment variable name, such as "AVIF_TEST_TMP_DIR". That will require passing the environment variable name as a command-line argument to avify4mtest.c. What do you think?

Comment (y-guyon) [reply tests/avify4mtest.c:163]: Yes, passing the environment variable name as a command-line argument to avify4mtest is a good idea.

Comment (wantehchang) [reply tests/avify4mtest.c]: Done.

File: tests/avify4mtest.c [hunk 2]
```diff
@@ -161,11 +161,16 @@ static avifBool encodeDecodeY4m(uint32_t width,
 int main(int argc, char * argv[])
 {
     if (argc != 2 || !strlen(argv[1])) {
-        fprintf(stderr, "Missing temporary folder path argument\n");
+        fprintf(stderr, "Missing temporary directory environment variable argument\n");
```
Comment (y-guyon) [inline tests/avify4mtest.c:164]: "Missing temporary directory path environment variable name argument"?

**Review by y-guyon — APPROVED** [review]

File: tests/avify4mtest.c (hunk 2 above)
Comment (wantehchang) [reply tests/avify4mtest.c]: Done. The error message is now precise but a little hard to parse.

**Pull request merged**